src/main/java/com/shaft/gui/element/JavaScriptWaitManager.java

```diff
@@ -10,13 +10,15 @@
 import org.openqa.selenium.support.ui.ExpectedCondition;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
+import java.time.Duration;
 import java.util.Objects;
 
 public class JavaScriptWaitManager {
     private static final int WAIT_DURATION_INTEGER = 15;
     private static final String TARGET_DOCUMENT_READY_STATE = "complete";
     private static final ThreadLocal<WebDriver> jsWaitDriver = new ThreadLocal<>();
     private static final int delayBetweenPolls = 20; // milliseconds
+    private static final Duration WAIT_DURATION = Duration.ofSeconds(WAIT_DURATION_INTEGER);
```

Comment:
Codacy found an issue: [Avoid unused private fields such as 'WAIT_DURATION'.](https://app.codacy.com/manual/mohab.mohieeldeen/SHAFT_ENGINE/pullRequest?prid=5771854)

Reply:
in place for easy upgrade to selenium 4.x